<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-NM05SVP" ignored="0" name="Portfolio Test" tests="2" failures="1" timestamp="11 Apr 2018 10:35:38 GMT" time="227.455" errors="0">
  <testcase name="buyOrSellStockTest" time="144.757" classname="com.qtpselenium.rediff.hybrid.testcases.stocksuite.BuySellStockTest"/>
  <testcase name="checkTransactionHistoryTest" time="80.767" classname="com.qtpselenium.rediff.hybrid.testcases.stocksuite.BuySellStockTest">
    <failure type="java.lang.reflect.InvocationTargetException">
      <![CDATA[java.lang.reflect.InvocationTargetException
	at com.qtpselenium.rediff.hybrid.driver.DriverScript.executeKeywords(DriverScript.java:47)
	at com.qtpselenium.rediff.hybrid.testcases.stocksuite.BuySellStockTest.checkTransactionHistoryTest(BuySellStockTest.java:34)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.AssertionError: The following asserts failed:
	Actual average is 0.00Expected was 38.775510204081634
	at com.qtpselenium.rediff.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:348)
	at com.qtpselenium.rediff.hybrid.keywords.AppKeywords.checkHistory(AppKeywords.java:162)
	... 26 more
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- checkTransactionHistoryTest -->
</testsuite> <!-- Portfolio Test -->
